.github/workflows/release_prepare.yml

```diff
@@ -70,11 +70,30 @@ jobs:
                   fetch-tags: true
                   token: ${{ secrets.ELEMENT_BOT_TOKEN }}
 
-            - name: Merge develop
+            - name: Merge Element Desktop
+              if: inputs.element-desktop
+              run: |
+                  git config --global user.email "releases@riot.im"
+                  git config --global user.name "RiotRobot"
+                  git -C "element-desktop" merge origin/develop
+            - name: Merge Element Web
+              if: inputs.element-web
+              run: |
+                  git config --global user.email "releases@riot.im"
+                  git config --global user.name "RiotRobot"
+                  git -C "element-web" merge origin/develop
+            - name: Merge React SDK
+              if: inputs.matrix-react-sdk
+              run: |
+                  git config --global user.email "releases@riot.im"
+                  git config --global user.name "RiotRobot"
```

Comment:
Surely these don't need re-doing in each one given they all happen in the same env

Reply:
Ah, I wasn't 100% sure but now you mention it, yeah, of course it does.